hs_access_control/models.py

```diff
@@ -1717,7 +1733,7 @@ def share_resource_with_group(self, this_resource, this_group, this_privilege):
         if not self.can_share_resource(this_resource, this_privilege):
             raise HSAccessException("User has insufficient sharing privilege over resource")
 
-        if self not in access_group.members.all():
+        if self not in access_group.members.all() and not self.user.is_superuser:
             raise HSAccessException("User is not a member of the group")
```

Comment:
HSAccessException message should be changed to something like "User is not a member of the group and not an admin" to better inform what is going on here.